Source/Core/DolphinQt2/GameList/TableProxyModel.cpp

```diff
@@ -0,0 +1,18 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinQt2/GameList/TableProxyModel.h"
+#include "DolphinQt2/GameList/GameListModel.h"
+
+TableProxyModel::TableProxyModel(QObject* parent) : QSortFilterProxyModel(parent)
+{
+  setSortCaseSensitivity(Qt::CaseInsensitive);
+  sort(GameListModel::COL_TITLE);
```

Comment:
Is this needed? The documentation says calling QTableView::setSortingEnabled (which is done in GameList) will do this automatically.